src/QuartzNET-DynamoDB.Tests/Integration/JobStore/TriggerResumeTests.cs

```diff
@@ -91,10 +93,36 @@ public void ResumeAll()
             Assert.Equal("Normal", triggerState2.ToString());
         }
 
+        #region IDisposable implementation
+
+        bool _disposedValue = false;
+
+        protected virtual void Dispose(bool disposing)
```

Comment:
worth getting all the test classes extend one that implements the Dispose method like so?